start.sh

```diff
@@ -1,14 +1,19 @@
 #! /bin/bash
 
-while ! mysql -h $MYSQL_DB_HOST -u root -p$MYSQL_ROOT_PASSWORD -e "SET GLOBAL sql_mode = '';"; do
+until mysqladmin ping -h $MYSQL_DB_HOST -u root -p$MYSQL_ROOT_PASSWORD ; do
     sleep 3
 done
 
 echo "Connected to db"
 
 if ! [[ -f "/var/www/html/$SITE_NAME/configuration.php" ]]; then
-  ~/.composer/vendor/bin/joomla site:download --www=/var/www/html --release=3.9 $SITE_NAME
-  ~/.composer/vendor/bin/joomla site:install --www=/var/www/html --mysql-login=root:$MYSQL_ROOT_PASSWORD --mysql-host=$MYSQL_DB_HOST --mysql-database=$MYSQL_DB_NAME $SITE_NAME
+  ~/.composer/vendor/bin/joomla site:create \
```

Comment:
In questa zona viene fatto qualcosa che genera questi logs:
```
sh: 1: sudo: not found
web_1  | sh: 1: sudo: not found
web_1  | sh: 1: unzip: not found
```
Riusciamo a sistemare questa questione?

Reply:
Risolto. Vedere commit a191c25da9b6783492856b86ca96c3a1b887689a

Reply:
@simonegazza domanda: facendo `2>/dev/null` non ci stiamo perdendo i logs di eventuali errori?